Cilsil.Test/TestRunManager.cs

```diff
@@ -198,7 +209,7 @@ public void RunInfer(string cfgJson, string tenvJson, out string stdout, out str
             foreach (var bug in inferReport)
             {
                 var severity = bug.Value<string>("severity");
```

Comment:
change severity to enum instead of string

Reply:
> change severity to enum instead of string

I think struct is more appropriate in this case? 
![image](https://user-images.githubusercontent.com/63893533/119895682-eba8bf80-bef2-11eb-969f-7b7759aa9d06.png)

Reply:
Done